app/views/recipe_foods/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p style="color: green"><%= notice %></p>
+
+<%= render @recipe_food %>
+
+<div>
+  <%= link_to "Edit this recipe food", edit_recipe_food_path(@recipe_food) %> |
+  <%= link_to "Back to recipe foods", recipe_foods_path %>
+
+  <%= button_to "Destroy this recipe food", @recipe_food, method: :delete %>
```

Comment:
- [ ] kindly ensure your design matches the design given, check the comparison below to see the differences 

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/218188604-d7881f4f-842d-4317-b830-7b648b5dc24a.png) |  ![image](https://user-images.githubusercontent.com/61408860/218188650-c08f3f05-7d84-4d05-a285-8ce720e6a9dd.png)